.enb/lib/page.js

```diff
@@ -0,0 +1,126 @@
+var techs = require('../techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl],
+        [techs.bem.depsOld],
+        [techs.bem.files]
+    ]);
+
+    // css
+    node.addTechs([
+        [techs.css.stylusWithAutoprefixer, { browsers : browsers }],
+        [techs.css.stylus, { target : '?.ie.css', sourceSuffixes : ['styl', 'ie.styl'] }],
+    ]);
+    node.mode('development', function() {
+        node.addTechs([
+            [techs.borschik, { source : '?.css', target : '_?.css', freeze : true, minify : false }],
+            [techs.borschik, { source : '?.ie.css', target : '_?.ie.css', freeze : true, minify : false }]
+        ]);
+    });
+    node.mode('production', function() {
+        node.addTechs([
+            [techs.borschik, { source : '?.css', target : '_?.css', freeze : true, tech : 'cleancss', minify : true }],
+            [techs.borschik, { source : '?.ie.css', target : '_?.ie.css', freeze : true, tech : 'cleancss', minify : true }],
+        ]);
+    });
+    node.addTargets(['_?.css', '_?.ie.css']);
+
+    // JavaScript for browsers
```

Comment:
@andrewblond Sure, just if you like it we can try to make configs as close as possible to this. I'm not insist, just probably we can.